### ISSUE TITLE
couldn't install

### ISSUE THREAD

**Issue by necktwi** [nicohaenggi/SafariBooks-Downloader#30]
```
$ node -v
v8.7.0
$ npm -v
5.4.2
$ npm install
npm WARN deprecated fs-promise@1.0.0: Use mz or fs-extra^3.0 with Promise Support
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN safaribooks-downloader@1.0.0 No repository field.

added 163 packages in 7.251s
$ safaribooks-downloader --version
-bash: safaribooks-downloader: command not found
```

**Comment by xevenheaven** [comment]
`npm install` didn't work for me either, but installing globally with `npm install -g` worked for me.
@necktwi you might wanna give it a try. 😄

**Comment by necktwi** [comment]
I already tried your suggestion but strangely it worked now!

**Comment by V3RT1AG0** [comment]
try npm safaribooks-downloader --version

**Comment by LingarajBagali** [comment]
You should just need to mention npm modules in PATH, then the command will be available globally.
`C:\Users\<username>\AppData\Roaming\npm`

**Comment by LingarajBagali** [comment]
> sorry for the confusion here, but running with `safaribooks-downloader` only works when the module is installed globally. I will modify the readme so this is more clear in the next release.

@nicohaenggi Yes, even then it was coming as 'not recognized' which is why i had to put them in path variable under user's. Not sure if you wanted us to do the same according to README. But, i think it would make sense if you mention it exclusively.
